### ISSUE TITLE
fix: SEO

### ISSUE THREAD

**Issue by schollz** [schollz/share#55]
- Document does not have a valid rel=canonicalPoints to the domain's root URL (the homepage), instead of an equivalent page of content
- robots.txt is not valid 
- Background and foreground colors do not have a sufficient contrast ratio.:div.font-black.uppercase.text-sm.sm:text-base.text-gray-600 AND div.border-2.sm:border-4.border-black.p-6.sm:p-8.text-center.transition-all.bg-gra...
